app/routes/articles.js

```diff
@@ -221,9 +213,10 @@ router.put('/:link', checkArticleOwnership, validation(updateArticle, BODY), asy
         req.log('Article UPDATE Route:', req.body)
         return res.redirect('/')
     }
+
     try {
-        const article = await Article.findOneAndUpdate({link: req.params.link}, {$set: req.body}, {runValidators: true}).populate('author').exec()
-        if(article.title !== req.body.title) article.link = encodeURIComponent(req.body.title.replace(SPACES, DASH))
+        let article = await Article.findOneAndUpdate({link: req.params.link}, {$set: req.body}).exec()
```

Comment:
Shouldn't we try...catch this?

Reply:
It is in one:
![image](https://user-images.githubusercontent.com/15958401/105389303-980a2a80-5c0f-11eb-928b-08947742f7ad.png)